src/table/actionHeaderFactory.js

```diff
@@ -91,7 +91,8 @@ function actionFactory(action, actionConfiguration, schema) {
 
 function actionColumnFrom(
   { action, icon, text, dropDownAction, actionConfiguration = false },
-  schema
+  schema,
+  renderCloseDropDownAction
```

Comment:
this should probably be forceReRenderTable. And please add comments as to why we had to use this.

Reply:
Naming changed and added comments